tests/temp_env_var.py

```diff
@@ -0,0 +1,2 @@
+TEMP_ENV_VARS = {"PROMETHEUS_MULTIPROC_DIR": ".tox/.tmp/wsgi"}
```

Comment:
Instead of configuring pytest to set this environment variable, I think we can use the tox to prepare the test environment.

- Make the directory in `commands_pre`
- Set `PROMETHEUS_MULTIPROC_DIR` in `setenv`
- Use the tox substitutions instead of hardcoding the `.tox/`
- Optionally to delete the directory, it depends on where it is created.

Reply:
That's nice!  A bit cleaner!